src/extension/features/budget/budget-spending-goal/index.css

```diff
@@ -0,0 +1,23 @@
+input.spend-goal {
```

Comment:
I think these are all classes that the toolkit create rather than classes that YNAB creates, if that's the case, can we prefix them with `tk-`?

`tk-spend-goal`, `tk-goal-label`, `tk-percent-label`